### ISSUE TITLE
训练程序报错

### ISSUE THREAD

**Issue by gz7seven** [ShaohonChen/Qwen3-SmVL#13]
您好！

在执行train.py时，我的程序报错了，具体如下：

  File "/home/gz/projects/LLM/Qwen3-SmVL-main/Qwen3-SmVL-main/train.py", line 310, in <module>
    main(training_args)
  File "/home/gz/projects/LLM/Qwen3-SmVL-main/Qwen3-SmVL-main/train.py", line 232, in main
    trainer.train(resume_from_checkpoint=last_checkpoint)
  File "/home/gz/miniconda3/envs/LLM/lib/python3.10/site-packages/transformers/trainer.py", line 2237, in train
    return inner_training_loop(
  File "/home/gz/miniconda3/envs/LLM/lib/python3.10/site-packages/transformers/trainer.py", line 2532, in _inner_training_loop
    batch_samples, num_items_in_batch = self.get_batch_samples(epoch_iterator, num_batches, args.device)
  File "/home/gz/miniconda3/envs/LLM/lib/python3.10/site-packages/transformers/trainer.py", line 5343, in get_batch_samples
    batch_samples.append(next(epoch_iterator))
  File "/home/gz/miniconda3/envs/LLM/lib/python3.10/site-packages/accelerate/data_loader.py", line 567, in __iter__
    current_batch = next(dataloader_iter)
  File "/home/gz/miniconda3/envs/LLM/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 708, in __next__
    data = self._next_data()
  File "/home/gz/miniconda3/envs/LLM/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 764, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/home/gz/miniconda3/envs/LLM/lib/python3.10/site-packages/torch/utils/data/_utils/fetch.py", line 55, in fetch
    return self.collate_fn(data)
  File "/home/gz/projects/LLM/Qwen3-SmVL-main/Qwen3-SmVL-main/train.py", line 135, in data_collate_fix2k
    batch = processor(
  File "/home/gz/miniconda3/envs/LLM/lib/python3.10/site-packages/transformers/models/smolvlm/processing_smolvlm.py", line 345, in __call__
    text, vision_inputs = self.process_vision(
  File "/home/gz/miniconda3/envs/LLM/lib/python3.10/site-packages/transformers/models/smolvlm/processing_smolvlm.py", line 182, in process_vision
    image_inputs = self.image_processor(images, **output_kwargs["images_kwargs"])
  File "/home/gz/miniconda3/envs/LLM/lib/python3.10/site-packages/transformers/image_processing_utils.py", line 44, in __call__
    return self.preprocess(images, **kwargs)
  File "/home/gz/miniconda3/envs/LLM/lib/python3.10/site-packages/transformers/models/smolvlm/image_processing_smolvlm.py", line 843, in preprocess
    encoding = BatchFeature(data=data, tensor_type=return_tensors)
  File "/home/gz/miniconda3/envs/LLM/lib/python3.10/site-packages/transformers/feature_extraction_utils.py", line 79, in __init__
    self.convert_to_tensors(tensor_type=tensor_type)
  File "/home/gz/miniconda3/envs/LLM/lib/python3.10/site-packages/transformers/feature_extraction_utils.py", line 195, in convert_to_tensors
    raise ValueError(
Unable to create tensor, you should probably activate padding with 'padding=True' to have batched tensors with the same length.
  0%|                                                                                                                                         | 0/34668 [00:01<?, ?it/s]

有没有人遇到过，帮忙解决一下。

**Comment by mrydtA** [comment]
我也是这个错误  搞了一天了  还是没解决

**Comment by lovelyyoshino** [comment]
这些我都修复了，可以看看
